src/main/java/org/openmaptiles/layers/Transportation.java

```diff
@@ -362,13 +442,11 @@ public void process(Tables.OsmHighwayLinestring element, FeatureCollector featur
         .setAttr(Fields.CLASS, highwayClass)
         .setAttr(Fields.SUBCLASS, highwaySubclass(highwayClass, element.publicTransport(), highway))
         .setAttr(Fields.NETWORK, networkType != null ? networkType.name : null)
-        // TODO: including brunnel at low zooms leads to some large 300-400+kb z4-7 tiles, instead
-        //       we should only set brunnel if the line is above a certain length
-        .setAttr(Fields.BRUNNEL, brunnel(element.isBridge(), element.isTunnel(), element.isFord()))
+        .setAttrWithMinzoom(Fields.BRUNNEL, brunnelValue, brunnelMinzoom)
```

Comment:
FYI @phanecak-maptiler I implemented setAttrWithMinzoom in https://github.com/onthegomap/planetiler/pull/725 - you should be able to do this now:

```java
.setAttrWithMinSize(Fields.BRUNNEL, brunnelValue, 3, 4, 12) // min size=3px starting at z4, then always show at z12 and above
```